src/geometry.F90

```diff
@@ -129,8 +129,8 @@ recursive subroutine find_cell(p, found, search_cells)
     type(Particle), intent(inout) :: p
     logical,        intent(inout) :: found
     integer,        optional      :: search_cells(:)
-
```

Comment:
I would prefer that you leave the blank space here to separate dummy variables from regular variables.